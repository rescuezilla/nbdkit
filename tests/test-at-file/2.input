# Test case for tests/test-at-file.sh
# Naked parameters should be prefixed with "magic=".
1
b=2
3
d=4
