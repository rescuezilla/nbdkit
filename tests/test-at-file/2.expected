magic=1
b=2
magic=3
d=4
